src/modules/dashboard/components/ActionsPage/ActionsComponents/MintTokenMotion.tsx

```diff
@@ -172,6 +175,21 @@ const MintTokenMotion = ({
           />
         </div>
         <div className={styles.details}>
+          {motionState !== MotionState.StakeRequired && (
```

Comment:
Current `motionState` logic has an error.

Please cherry pick this commit: https://github.com/JoinColony/colonyDapp/pull/2447/commits/907459131a342d2a312bd2995ced8a45ffd08ee6

From my PR that fixes it